src/utils/notificationUtils.js

```diff
@@ -1,26 +1,29 @@
+import { deriveIndividualName } from './nameUtils';
+
 export const getNotificationProps = notification => {
   const senderName = notification?.sender_name || 'Unnamed User';
   const user1Name =
     notification?.message_values?.user1_name || 'Unnamed User';
   const user2Name =
     notification?.message_values?.user2_name || 'Unnamed User';
-  const individual1Names = notification?.names || []; // TODO flesh out more once this is included in notifications DEX-739
-  const individual1NicknameObject = individual1Names.find(
-    n => n.context === 'nickname',
+
+  // TODO flesh out more once this is included in notifications DEX-739
+  const individual1AdoptionName = deriveIndividualName(
```

Comment:
thank you for making this use deriveIndividualName

Reply:
The blocker for DEX-739 appears to have been removed, so at some point we should get back to this and not set both names to the same value. 😆